bin/i18n/resources/pegasus/emails/sync_in.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/pegasus_markdown'
+require_relative '../emails'
+
+module I18n
+  module Resources
+    module Pegasus
+      module Emails
+        class SyncIn < I18n::Utils::SyncInBase
+          LOCALIZABLE_FILE_SUBPATHS = %w[
+            hoc_signup_2023_receipt_en.md
+          ].freeze
+
+          def execute
+            progress_bar.start
+
+            I18nScriptUtils.process_in_threads(LOCALIZABLE_FILE_SUBPATHS) do |file_subpath|
```

Comment:
I'm a bit doubtful about whether we should process the emails in batches or not.
On one hand, we currently are adding just a single email, so processing in threads won't do anything at all.
On the other hand, if the number of emails grows, processing in threads will keep the speed optimized.
I'd like to hear @artem-vavilov opinion on this.